dts/bindings/sensor/invensense,icm4268x-common.yaml

```diff
@@ -0,0 +1,318 @@
+# Copyright (c) 2024 Intel Corporation
+# Copyright (c) 2022 Esco Medical ApS
+# Copyright (c) 2020 TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    ICM-4268X motion tracking device
+compatible: "invensense,icm4268x-common"
+include: [sensor-device.yaml, spi-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: |
+      The INT signal default configuration is active-high. The
+      property value should ensure the flags properly describe the
+      signal that is presented to the driver.
+
+  int1-pol:
+    type: int
+    default: 0
```

Comment:
looks like this should not have a default but maybe be required: true. otherwise, justify the default